packages/react/src/components/Accordion/docs/overview.mdx

```diff
@@ -3,10 +3,6 @@
 <StorybookDemo
   themeSelector
   url="https://react.carbondesignsystem.com"
-  variants={[
-    {
-      label: 'Default',
-      variant: 'components-accordion--default'
-    }
-  ]}
-/>
+>
+  <Variant label="Default" variant="components-accordion--default" />
+</StorybookDemo>
```

Comment:
@francinelucca this new API looks so nice!
What are your thoughts on the prop name of `variant` since the component is named also named `Variant`? Should we find a more specific name for the prop?

Reply:
should we name it `path` instead? 'cause it is the path the variant lives in inside the `StorybookDemo` `url`

Reply:
@francinelucca You might consider `id` or a more descriptive `storyId`. As a reference, [our e2e tests](https://github.com/carbon-design-system/carbon/blob/b53e3e9e02a18e4cf94b9f90e31caf4e979ac05b/e2e/test-utils/storybook.js#L30-L31) visit stories via url like this and we mirror the pattern that storybook itself references/names these as.

`storyId` (or simply `id`) consists of a `component` and `story`:

```js
const component = `accordion`;
const story = `default`;
const storyId = `components-${component}--${story}`;
```

Reply:
I like `storyId` for the prop name.

Reply:
same! I think we should go with `storyId` :)